source/Music.h

```diff
@@ -40,6 +40,8 @@ class Music {
 	void SetSource(const std::string &name = "");
 	const std::vector<int16_t> &NextChunk();
 
+	std::string trackName = "";
```

Comment:
Make this variable private and then add a `GetSource()` function to get the source. Doing it like this is not great since anyone can change the track name.

Reply:
https://github.com/endless-sky/endless-sky/blob/27584823949977685fb2f7299408078c87a8af6c/source/Music.cpp#L32 this is the only location track names are stored.  So that's why we need to store the current track name as a string to avoid having to walk through a hashmap.

Per our discussion on discord I think this is resolved for now.